akka-projection-core/src/main/java/akka/projection/internal/spi/ProjectionMetrics.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.internal.spi;
+
+import akka.actor.ClassicActorSystemProvider;
+
+public interface ProjectionMetrics {
```

Comment:
no reason to write this in Java as far as I can see. no difficult signatures here